script.js

```diff
@@ -158,6 +178,50 @@ const createListItem = (location, status, lng, lat) => {
   return $item
 }
 
+//////////////////////////
+// Protect against columns not yet existing in the spreadsheet.
+// We can remove once they are added to the sheet.
+function extractSeekingMoney(item) {
+  try {
+    return truthy(item.gsx$seekingmoney.$t);
```

Comment:
```suggestion
    const moneySearchStr = `${item.gsx$accepting.$t}, ${item.gsx$urgentneed.$t}, ${item.gsx$notes.$t}`.toLowerCase()
    const legacySeekingMoney = moneySearchStr.includes('money') || moneySearchStr.includes('cash') || moneySearchStr.includes('monetary')
    return truthy(item.gsx$seekingmoney.$t) || legacySeekingMoney
```

I think we might need the old button behavior too. Probably need to talk to the organizers though.

Reply:
I did not add this. Looking at the spreadsheet it appears the 'seekingMoney' column is being used correctly.